dali/base/dasess.cpp

```diff
@@ -648,8 +625,13 @@ class CSessionRequestServer: public Thread
                 StringBuffer reqSignature;
                 CDateTime reqUTCTimestamp;
                 if (queryDaliServerVersion().compare("3.15") >= 0)
```

Comment:
Why is there a test on the server version in the code?  This is executed server side, so it should always be true.

Reply:
Good point, removed...